### PR TITLE
Fix quotation issue when using REPORTDIR

### PR DESCRIPTION
**Pull request by renfeiw**
- created another variable REPORTDIR_NQ
- use it inside quoation

[ci skip]

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
reviewer: @llxia

**Comment by smlambert** [comment]
I wonder if we should tag these PRs that are being done to be able to support running tests in both open projects & testing in Jenkins and the in-house/closed proprietary system (so they are easier to find and remove later, for changes that not needed for Jenkins testing).  

I heard from @fjeremic that we are targeting end of year to try and pull away from in-house proprietary system.  Is it true?  Can we get that into a schedule and a set of real goals to target?

This is good news if true, as supporting 2 disparate systems costs time and introduces compromises that would be preferable to avoid.

File: test/TestConfig/settings.mk [hunk 1]
```diff
@@ -212,9 +212,11 @@ ifndef UNIQUEID
 endif
 TESTOUTPUT := $(TEST_ROOT)$(D)TestConfig$(D)test_output_$(UNIQUEID)
 ifeq ($(TEST_ITERATIONS), 1)
-	REPORTDIR = $(Q)$(TESTOUTPUT)$(D)$@$(Q)
+	REPORTDIR_NQ = $(TESTOUTPUT)$(D)$@
+	REPORTDIR = $(Q)$(REPORTDIR_NQ)$(Q)
 else
-	REPORTDIR = $(Q)$(TESTOUTPUT)$(D)$@_ITER_$$itercnt$(Q)
+	REPORTDIR_NQ = $(TESTOUTPUT)$(D)$@_ITER_$$itercnt
+	REPORTDIR = $(Q)$(REPORTDIR_NQ)$(Q)
```
Comment (llxia) [inline test/TestConfig/settings.mk]: Should we set `REPORTDIR = $(Q)$(REPORTDIR_NQ)$(Q)` after if else?

**Comment by fjeremic** [comment]
> I heard from @fjeremic that we are targeting end of year to try and pull away from in-house proprietary system. Is it true? Can we get that into a schedule and a set of real goals to target?

I think so. There are several internal issues opened to make this happen. There is an Epic _Migrate IBM Java8 Build from Axxon to Jenkins_ tracking this work. It was mentioned on an infrastructure call either by @jdekonin or @AdamBrousseau in the last few months.

File: test/TestConfig/settings.mk (hunk 1 above)
Comment (renfeiw) [reply test/TestConfig/settings.mk]: fixed

**Comment by llxia** [comment]
jenkins test sanity plinux,win jdk8

**Pull request merged**